biodatacatalyst-ui/src/main/webapp/picsureui/search-interface/filter-model.js

```diff
@@ -202,7 +202,7 @@ define(["backbone", "handlebars", "picSure/settings", "picSure/queryBuilder", "o
             generateDatatableCategory: function(searchResult) {
                 return "\\" + searchResult.result.dtId + "\\" + searchResult.result.studyId + "\\";
             },
-            addExportColumn: function(searchResult, type, source){
+            addExportColumn: function(searchResult, type, source, selectedValues, selectedMin, selectedMax){
```

Comment:
`selectedValues` is always `undefined` when you call it above. Is it getting populated in a different part of the code? If not, you might not need to pass it here, and could just put some default value below.